johnsonandjohnson/Bodiless-JS
# fix(vital-demo): Add Homepage Content & minor fixes

### Changes

Content: 

- Added Content to Homepage
- Fixed Styleguide link in menu
- Fixed card link on /styleguide page.

Code: 

- Restricted Topslot on generic template to constrain to 2xl:  
- Moved breadcrumb styling to correct place (was a todo) so it takes effect.

### Testing:

- Verify Homepage Content
- Verify Styleguide link in menu
- Verify card link on /styleguide page
- Verify if browser is open larger than 1536 than top slot is contained to 1536.)
- Verify breadcrumbs should get the font styles: text-vital-primary-body-copy font-normal text-m-xs lg:text-xs
